sickbeard/providers/torrent/html/transmitthenet.py

```diff
@@ -148,13 +148,13 @@ def parse(self, data, mode):
                     download_url = urljoin(self.url, download_item['href'])
 
                     temp_anchor = row.find('a', {'data-src': True})
-                    title = temp_anchor['data-src'].rsplit('.', 1)[0]
```

Comment:
This was causing for stuff from beeing removed from release names, not all are scene.